components/supervisor/frontend/src/index.ts

```diff
@@ -11,6 +11,13 @@
 import { IDEMetricsServiceClient, MetricsName } from "./ide/ide-metrics-service-client";
 IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendClientTotal).catch(() => {})
 
+try {
```

Comment:
We surround this whole thing in a `try...catch` block, because `sessionStorage` may fail due to some browser constraints. If it does, it should be alright to just skip reporting here.

MDN has more details about why this should be done and when this can happen [[1](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#testing_for_availability)]